### PR TITLE
Community folder

### PR DESCRIPTION
**Pull request by ghost**
**Description**
Added new folder `community` and added NekosBest to it

### CODE REVIEW

**Comment by TheCataliasTNT2k** [comment]
Successor of https://github.com/PyDrocsid/cogs/pull/166

File: community/nekosbest/cog.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    CONTRIBUTORS = [Contributor.NekoFanatic]
+
+    @commands.command(name="nekos")
+    @docs(t.commands.neko)
+    async def neko(self, ctx: Context):
+        async with ClientSession() as session:
+            resp = await session.get("https://nekos.best/api/v1/nekos")
+            nekos = await resp.json()
+
+            embed = Embed(title="Sauce", url=nekos["source_url"], color=Colors.NekosBest)
```
Comment (Tert0) [inline community/nekosbest/cog.py]: Please use the Translation for the Embed Title

File: community/nekosbest/cog.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+from aiohttp import ClientSession
+
+tg = t.g
+t = t.nekosbest
+
+
+class NekosBestCog(Cog, name="NekosBest"):
+    CONTRIBUTORS = [Contributor.NekoFanatic]
+
+    @commands.command(name="nekos")
```
Comment (Defelo) [inline community/nekosbest/cog.py]:
```suggestion
    @commands.command()
```

**Review by Defelo — CHANGES_REQUESTED** [review]

File: community/nekosbest/cog.py (hunk 1 above)
Comment (ghost) [reply community/nekosbest/cog.py]: done

File: community/nekosbest/cog.py (hunk 2 above)
Comment (ghost) [reply community/nekosbest/cog.py]: done

**Review by Defelo — CHANGES_REQUESTED** [review]
Please don't mix PRs! Commit 083c9aae1da0821484a96f4dbf04c01256a1e973 is not supposed to be here!

**Comment by ghost** [comment]
@Defelo that was not intented, github used this pr, idk why. i tried to change it but he still tried to connect it to this pr

**Pull request closed without merging**